src/docs/inclusion.mdx

```diff
@@ -0,0 +1,86 @@
+something in the code base that is hard to get rid of.
+
+As such this document largely determines what should be done for new code, but
+not necessarily for existing code.
+
+## Phased Out Terms
+
+We have identified some terms that we no longer want to use going forward but
+that might still exist in parts of the code base.
+
+master/slave:
+
+: Generally the terms to be used in the context of databases should be
+  "primary"/"secondary" or "primary"/"replica".
+
+whitelist/blacklist:
+
+: Generally should not be used.  We try to avoid the word "list" now in general
+  and try to be more descriptive in the UI.  For instance the use of
+  "ip blacklist" has been changed to "ip filters" as part of the filter product.
+  If you need a generic term "allowlist" and "denylist" or "blocklist" are
+  good substitutions.
+
+whitelisted/blacklisted:
+
+: Try to avoid generic verbs here and better describe directly what it is
+  you're trying to do.  For instance we changed "whitelisted environment
+  variables" to "safe environment variables".
+
+
+## Acceptable Terms
+
+master:
+
+: Master in isolation is acceptable if the origin of the term can exclude the
+  associated meaning of "slave".  This for instance is the case when this is
+  referred to as a "master copy", a "master degree" etc.  For the git branch
+  name please see below.
+
+
+## Git Branch Naming
+
+We're okay keeping the term "master" for existing repositories but it's
+generally recommended to change the default branch name to "main" going forward
+on new repositories.  The term "master" in isolation is acceptable in general
+however the history of the branch name in git comes from bitkeeper where this
+terminology referred to master and slave repositories.  At present changing the
+default branch on existing repositories has negative consequences for users
+that have already clonsed the repositories so these should not be changed.
+
+To reconfigure git to use a different default repo for `git init` a template
+can be configured.  Put the following into your `~/.gitconfig`:
+
+```ini {tabTitle: Git Config}
+[init]
+	templateDir = ~/.config/git/template/
+```
+
+And then create an appropriate template like this:
+
+```bash
+mkdir -p "$(git config --path --get init.templateDir)"
+echo "ref: refs/heads/main" > "$(git config --path --get init.templateDir)/HEAD"
+```
+
+From this moment onwards future repositories will use the default branch name "main" instead.
+
+## Changing Code
+
+If you identified code or comments that use problematic language you are free
+to change it.  However please be advised that some uses of problematic language
+haven't been phased out because they are backwards incomaptible changes and
+require a lot more work.  Specifically for instance the rate limiting code in
+Sentry uses the terms "whitelist" for persisted data and for consistency the
+code that deals with this data structure for now retains these uses.
```

Comment:
```suggestion
If you identified code or comments that use a problematic language you are free
to change it.  However please be advised that some uses of problematic language
have not been phased out because they are backward-incompatible changes and
require a lot more work.  Specifically, for instance, the rate-limiting code in
Sentry uses the terms "whitelist" for persisted data and for consistency the
code that deals with this data structure, for now, retains these uses.
```